src/utils/sorter.js

```diff
@@ -30,7 +30,7 @@ function isSortingNeeded(order?: Order[], sortBy: SortBy, sortDirection: SortDir
  * @param {Cache} cache item cache
  * @return {void}
  */
-export default function(
+export default function sorter(
```

Comment:
is this an eslint thing that's not allowing anonymous functions?

Reply:
yea its a rule that was added when i bump up eslint